<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="widgetFrameOuter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="class" stdset="0">
      <string>PeripheralCollection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_4">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>80</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="class" stdset="0">
            <string>PeripheralCollectionInner</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableWidget" name="valueTable">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>60</height>
               </size>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustIgnored</enum>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::NoSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="rowCount">
               <number>1</number>
              </property>
              <property name="columnCount">
               <number>16</number>
              </property>
              <attribute name="horizontalHeaderMinimumSectionSize">
               <number>55</number>
              </attribute>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>55</number>
              </attribute>
              <attribute name="horizontalHeaderHighlightSections">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <row/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="class" stdset="0">
            <string>PeripheralCollectionInner</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <property name="leftMargin">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <property name="horizontalSpacing">
             <number>2</number>
            </property>
            <item row="1" column="0" rowspan="3">
             <widget class="QScrollArea" name="scrollArea">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>957</width>
                 <height>593</height>
                </rect>
               </property>
               <layout class="QGridLayout" name="gridLayout_6">
                <property name="leftMargin">
                 <number>2</number>
                </property>
                <property name="topMargin">
                 <number>2</number>
                </property>
                <property name="rightMargin">
                 <number>2</number>
                </property>
                <property name="bottomMargin">
                 <number>2</number>
                </property>
                <property name="spacing">
                 <number>2</number>
                </property>
                <item row="17" column="0">
                 <widget class="QCheckBox" name="checkBox_10">
                  <property name="text">
                   <string>10</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QPushButton" name="pushButton_7">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Clear</string>
                  </property>
                 </widget>
                </item>
                <item row="20" column="0">
                 <widget class="QCheckBox" name="checkBox_13">
                  <property name="text">
                   <string>13</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="0">
                 <widget class="QCheckBox" name="checkBox_0">
                  <property name="text">
                   <string>0</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QSpinBox" name="durationBox">
                  <property name="suffix">
                   <string> S</string>
                  </property>
                  <property name="prefix">
                   <string>Duration </string>
                  </property>
                  <property name="value">
                   <number>20</number>
                  </property>
                 </widget>
                </item>
                <item row="15" column="0">
                 <widget class="QCheckBox" name="checkBox_8">
                  <property name="text">
                   <string>8</string>
                  </property>
                 </widget>
                </item>
                <item row="11" column="0">
                 <widget class="QCheckBox" name="checkBox_4">
                  <property name="text">
                   <string>4</string>
                  </property>
                 </widget>
                </item>
                <item row="14" column="0">
                 <widget class="QCheckBox" name="checkBox_7">
                  <property name="text">
                   <string>7</string>
                  </property>
                 </widget>
                </item>
                <item row="16" column="0">
                 <widget class="QCheckBox" name="checkBox_9">
                  <property name="text">
                   <string>9</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QPushButton" name="pushButton_4">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Start</string>
                  </property>
                 </widget>
                </item>
                <item row="18" column="0">
                 <widget class="QCheckBox" name="checkBox_11">
                  <property name="text">
                   <string>11</string>
                  </property>
                 </widget>
                </item>
                <item row="21" column="0">
                 <widget class="QCheckBox" name="checkBox_14">
                  <property name="text">
                   <string>14</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QPushButton" name="pushButton_6">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Save Data</string>
                  </property>
                 </widget>
                </item>
                <item row="12" column="0">
                 <widget class="QCheckBox" name="checkBox_5">
                  <property name="text">
                   <string>5</string>
                  </property>
                 </widget>
                </item>
                <item row="8" column="0">
                 <widget class="QCheckBox" name="checkBox_1">
                  <property name="text">
                   <string>1</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="22" column="0">
                 <widget class="QCheckBox" name="checkBox_15">
                  <property name="text">
                   <string>15</string>
                  </property>
                 </widget>
                </item>
                <item row="10" column="0">
                 <widget class="QCheckBox" name="checkBox_3">
                  <property name="text">
                   <string>3</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QPushButton" name="pushButton_5">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Stop</string>
                  </property>
                 </widget>
                </item>
                <item row="19" column="0">
                 <widget class="QCheckBox" name="checkBox_12">
                  <property name="text">
                   <string>12</string>
                  </property>
                 </widget>
                </item>
                <item row="13" column="0">
                 <widget class="QCheckBox" name="checkBox_6">
                  <property name="text">
                   <string>6</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QSpinBox" name="settlingBox">
                  <property name="suffix">
                   <string>mS</string>
                  </property>
                  <property name="prefix">
                   <string>Settling Delay </string>
                  </property>
                  <property name="value">
                   <number>5</number>
                  </property>
                 </widget>
                </item>
                <item row="9" column="0">
                 <widget class="QCheckBox" name="checkBox_2">
                  <property name="text">
                   <string>2</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QComboBox" name="comboBox">
                  <item>
                   <property name="text">
                    <string>A1</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>A2</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>A3</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>ADS1115_UNI_0</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>ADS1115_UNI_1</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>ADS1115_UNI_2</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>ADS1115_UNI_3</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>ADS1115_DIFF_01</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>ADS1115_DIFF_23</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="0" column="1" rowspan="24">
                 <layout class="QGridLayout" name="plot_area"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="msgwin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{color:green}</string>
              </property>
              <property name="text">
               <string>.</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>startLogging()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>stopLogging()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>clearTraces()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>saveTraces()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>setADC(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>-29</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startLogging()</slot>
  <slot>stopLogging()</slot>
  <slot>clearTraces()</slot>
  <slot>saveTraces()</slot>
  <slot>slot1()</slot>
  <slot>setADC(QString)</slot>
 </slots>
</ui>
